sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -248,6 +250,19 @@ internal class RadarLocationManager(
         }
     }
 
+    internal fun updateTrackingFromMeta(meta: RadarMeta?) {
+        if (meta?.remoteTrackingOptions != null) {
+            // use remotely-configured options if specified
+            RadarSettings.setTrackingOptions(context, meta.remoteTrackingOptions)
+            RadarSettings.setShouldListenToServerTrackingOptions(context, true)
+        } else {
+            // fallback
+            RadarSettings.revertToFallbackTrackingOptions(context)
```

Comment:
instead of "fallback", I think we should be more explicit and call it client options

Reply:
cc @jkao - can you weigh in on your naming preference?

Reply:
just a heads up (and my bad for not doing this implementation to begin with) -
we've actually changed the implementation of this on iOS: https://github.com/radarlabs/radar-sdk-ios/pull/127/files#diff-4689395a38be45a14ce3f56437a52efe930adc0d3cf9f644915693ab811efa04R365-R380

instead of doing the fallback, we have a new remote-tracking-options key that is populated if we see it in meta, and it gets deleted otherwise.

we have then changed `Radar.getTrackingOptions` to return `RadarSettings.remoteTrackingOptions || RadarSettings.trackingOptions`

Reply:
cc @jrtibbetts

Reply:
Done - updated this PR following the iOS workflow.